browser/resources/settings/brave_tor_page/brave_tor_bridges_dialog.ts

```diff
@@ -67,20 +74,19 @@ class RequestBridgesDialog extends RequestBridgesDialogBase {
     this.captchaResolve_ = ''
     this.status_ = this.i18n('torRequestBridgeDialogWaiting')
     this.captcha_ = ''
-    this.enableSubmit_(false)
+
     this.browserProxy_.requestBridgesCaptcha().then((result) => {
       this.captcha_ = result.captcha
       this.status_ = this.i18n('torRequestBridgeDialogSolve')
-      this.enableSubmit_(true)
+      this.renewDisabled_ = false;
     }, () => {
       this.status_ = this.i18n('torRequestBridgeDialogError')
       this.renewDisabled_ = false
     })
   }
 
-  enableSubmit_(enabled) {
-    this.renewDisabled_ = !enabled
-    this.submitDisabled_ = !enabled
+  computeSubmitDisabled_(captchaResolve) {
```

Comment:
Please define the type for the `captchaResolve` param

Reply:
defined as String